src/get_functions.jl

```diff
@@ -0,0 +1,48 @@
+"""
+    get_origin( vtk_file )
```

Comment:
```suggestion
    get_origin(vtk_file)
```
unnecessary whitespace